docs/UserGuide.md

```diff
@@ -58,121 +68,131 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
+<!-- ### Viewing help : `help`
 
 Shows a message explaning how to access the help page.
 
 ![help message](images/helpMessage.png)
 
-Format: `help`
+Format: `help` -->
 
+### Creating a project: `project`
 
-### Adding a person: `add`
+Create a project to be tracked by the application.
 
-Adds a person to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `project n/NAME d/DESCRIPTION [t/TAG]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+Example: `project n/Momentum d/CS2103T Team Project t/impt`
+
+Result: Creates a project named “Momentum” with a description “CS2103T Team Project” and a tag "impt".
+
+### Updating a project: `edit`
```

Comment:
Not sure if it would be better to rename this as 'Editing a project'? It might be more clear that the name of the feature is the same as the command.

Reply:
Changed to edit